discretenet/problems/school_bus_scheduling/school_bus_scheduling.py

```diff
@@ -0,0 +1,185 @@
+import numpy.random as random
```

Comment:
This is somewhat confusing with the standard library `random` module. Could you instead do the normal `import numpy as np` and use `np.random?`

Also with the note above about improt order, can you move this to below the pathlib and typing imports with a newline between them:

```python
from pathlib import Path
from typing import Union

import numpy as np
import pyomo.environ as pyo

from discretenet.problem import Problem
from discretenet.generator import Generator
```

Reply:
Updated